tests/Proto.Actor.Tests/SupervisionTests_OneForOne.cs

```diff
@@ -214,5 +214,21 @@ public void OneForOneStrategy_Should_StopChildOnFailureWhenStarted()
             Assert.Contains(Stop.Instance, childMailboxStats.Posted);
             Assert.Contains(Stop.Instance, childMailboxStats.Received);
         }
+        
+        [Fact]
+        public void OneForOneStrategy_Should_SuspendParentMailboxOnEscalateFailure()
```

Comment:
Does this capture the intent @rogeralsing ?

Reply:
Yes, but I think `Restart` would probably be a more intuitive message than `SuspendMailbox` as the Suspend happens on escalation and Restart when the grandparent have actually restarted the parent